### ISSUE TITLE
Dot separator not centered in content, front and backend

### ISSUE THREAD

**Issue by ianbelanger79** [WordPress/twentynineteen#429]
When a Dot Separator is added, it is not centered within the content on the front or backend. See screenshots below.

Frontend
![dot-separator-not-centered](https://user-images.githubusercontent.com/34601694/47793140-f88db400-dcf3-11e8-9b1c-fd5b2cc363c1.PNG)

Backend
![dot-separator-not-centered-backend](https://user-images.githubusercontent.com/34601694/47793149-fd526800-dcf3-11e8-8a74-6c745636a5c8.PNG)

**Comment by anevins12** [comment]
Yikes I only fixed front-end

**Comment by kjellr** [comment]
> Yikes I only fixed front-end

No problem — I submitted a separate PR for the editor: #455. Thanks again!

**Comment by ianbelanger79** [comment]
Unfortunately #447 does not fix this issue. I just pulled in the latest commit and tested in Firefox and Chrome, here is what I am seeing

![429-dot-separator-not-centered](https://user-images.githubusercontent.com/34601694/47873098-b2fde380-dde6-11e8-83b4-0aa951de1604.PNG)

In the editor, the dots are centered or will be once #455 is merged.

**Comment by kjellr** [comment]
I think this should be fixed as of https://github.com/WordPress/twentynineteen/compare/2d6914dc16bf...33b4b0e494ba. @ianbelanger79 mind giving it another try?

**Comment by ianbelanger79** [comment]
@kjellr LGTM now, closing

**Comment by kjellr** [comment]
Thanks everyone!
